code/scripting/lua/LuaTable.h

```diff
@@ -19,7 +19,7 @@ class LuaTable;
  * be in the exact same state when you call the next method of this class.
  */
 class LuaTableIterator {
-	lua_State* _L = nullptr;
+	lua_State* L_State = nullptr;
```

Comment:
The rest of the class uses the format `_varname` so this field should do the same. `_luaState` would be an acceptable name.